.scripts/build_steps.sh

```diff
@@ -34,9 +34,15 @@ CONDARC
 export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 
 mamba install --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build boa conda-forge-ci-setup=3
+    pip mamba conda-build conda-forge-ci-setup=4 "conda-libmamba-solver>=23.11.1"
 mamba update --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build boa conda-forge-ci-setup=3
+    pip mamba conda-build conda-forge-ci-setup=4 "conda-libmamba-solver>=23.11.1"
+# TEMPORARY:
+pip install -U https://github.com/jaimergp/conda-build/archive/debug-cls-keyerror.tar.gz --no-deps
+pip install -U git+https://github.com/conda/conda-libmamba-solver.git@conda-build-offline --no-deps
+pip install -U https://github.com/jaimergp/conda/archive/investigate-cb-key-error.tar.gz --no-deps
```

Comment:
If this gives any trouble, you could also do the
```sh
python -c "import pathlib, conda.plan; src = pathlib.Path(conda.plan.__file__); text = src.read_text().replace('stack_context_default', 'stack_context'); src.unlink(); src.write_text(text)"
```
spiel.

Reply:
`investigate-cb-key-error` also has some extra logging in case we hit a KeyError, and does not raise so the build can proceed without the report. I think the patch does work (you can see the repodata not being re-downloaded). The error is now about `import lsst` in the tests 🤷

Reply:
Ah, I saw https://github.com/conda/conda-libmamba-solver/issues/393#issuecomment-1824126032. Ok, reverting and rerunning.